code/style.css

```diff
@@ -1,7 +1,347 @@
+  display: flex; 
+  justify-content: center;
+  flex-direction: column;
+  margin: 0px 20px 20px 40px;
+}
+
+/* FORM PART */
+
+.form-container {
+  background-color: #ede7e4;
+  /* box-sizing: border-box; */
+  padding: 10px;
+  color: black;
+  margin-bottom: 30px;
+  margin-right: 20px;
+  border-radius: 30px;
+}
+.header-form {
+
+  padding: 15px;
+  border-radius: 10px;
+  margin-bottom: 10px;
+  text-transform: uppercase;
+  font-size: 17px;
+  text-align: center;
+}
+.form-question {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 15px;
+  margin-left: 20px;
+}
+
+.input-text {
+  padding: 8px;
+  width: 90%;
+  background: #dacec9;
+  border-radius: 5px;
+  border: none;
+  margin-top: 4px;
+}
+.form-radio {
+  margin-bottom: 13px;
+  font-size: 15px;
+  display: block;
+  margin-left: 20px;
+}
+.type-coffee {
+  margin-bottom: 5px;
+}
+.how-aboutus {
+  margin-left: 20px;
+}
+select {
+  background-color: #dacec9;
+  color: grey;
+  margin-left: 15px;
+  border-radius: 5px;
+  cursor: pointer;
+}
+input:focus {
+  background: antiquewhite;
+}
+
+input[type="checkbox"] {
+  filter: invert(80%);
+}
+
+input[type="radio"] {
+  filter: invert(80%);
+}
+
+.form-checkbox {
+  margin-top: 40px;
+  text-align: center;
+}
+
+.submit-button {
+  display: flex;
+  justify-content: center;
+  margin-top: 12px;
+}
+.order-button {
+  background-color: black;
+  color: white;
+  cursor: wait;
+  padding: 10px 30px 10px 30px;
+  border-radius: 10px;
+}
+
+/* Accordion part */
+
+.header-accordion {
+  padding: 15px;
+  border-radius: 10px;
+  margin-bottom: 10px;
+  margin-top: 10px;
+  text-transform: uppercase;
+  font-size: 17px;
+  text-align: center;
+}
+.accordion-container {
+  background-color: #ede7e4;
+  border-radius: 30px;
+  margin-right: 20px;
+}
+.accordion-item {
+  background-color: #dacec9;
+  color: black;
+  margin: 10px 20px 10px 20px;
+  border-radius: 5px;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.4);
+}
+
+.accordion-item-header {
+  padding: 10px 10px 20px 10px;
+  min-height: 30px;
+  font-weight: bold;
+  display:flex;
+  align-items: center;
+  position: relative;
+  cursor: zoom-in;
+}
+
+.accordion-item-header::after {
+  content: "\002B";
+  font-size: 28px;
+  color: antiquewhite;
+  position: absolute;
+  right: 16px;
+  transition: transform 0.5s ease-in-out;
+}
+
+.accordion-item-header.active::after {
+  transform: rotate(45deg);
+}
+
+.accordion-item-body {
+  background-color: #ebe0de;
+  display: none;
+  padding: 10px;
+  line-height: 20px;
+  border-top: 0.5px solid #a38e8a;
+  border-radius: 5px ;
+}
+
 /* the + sign is used to select the sibling that comes immediately next to the element you selected.
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
+.accordion-item-header.active + .accordion-item-body {
```

Comment:
Wouldn't it work if it was only .active + .accordion-item-body ?

Reply:
I am actually not sure, will try it when I have time, but I wanted to secure that JS knows where to take the class from :)